### ISSUE TITLE
I think the library just broke completely

### ISSUE THREAD

**Issue by stefan-reich** [ShyykoSerhiy/skyweb#84]
I can't get it to log in anymore. "failed to find t inside" every time. Haven't changed anything in the code or how I use it. I log in on web.skype.com in Chrome, then start the bot with the same credentials. This used to work, but now it just doesn't.

**Comment by ShyykoSerhiy** [comment]
I'll have a look on 7th of August. No guarantees it can be fixed quick though as I don't know what Skype has changed in auth flow.

**Comment by stefan-reich** [comment]
Yeah I just realized it is essentially a webscraping project. Not sure why I didn't think it could break unexpectedly. We're moving to Discord, lol. Thanks for your library anyway, we did use it for a bit.
